DLaB.CrmSvcUtilExtensions/OptionSet/Transliteration/TransliterationAlphabet.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DLaB.CrmSvcUtilExtensions.OptionSet.Transliteration
+{
+    public class TransliterationAlphabet
+    {
+        public int LanguageCode { get; private set; }
+
+        public Dictionary<char, string> Alphabet { get; private set; }
+
+        public TransliterationAlphabet(int langageCode, Dictionary<char, string> transliterationDict)
+        {
+            this.LanguageCode = langageCode;
+
+            this.Alphabet = transliterationDict;
+        }
+
+        /// <summary>
+        /// Transliterate from Latin to specified aplhabet.
+        /// </summary>
+        /// <param name="text">Text to transliterate.</param>
+        /// <returns>Returns transliterated text.</returns>
+        public string Transliterate(string text)
+        {
+            if (string.IsNullOrEmpty(text))
+            {
+                return "";
+            }
+
+            var result = "";
```

Comment:
This is highly inefficient, not sure what impact it would have, but if you made `result` a `List<String>` and then did a `string.Join(result)` as the return, it would alleviate having to do all the string concatenations.